<launch>
  <arg name="controller" default="spacemouse" />

  <rosparam command="load" ns="moveit_servo" file="$(find tahoma_moveit_config)/config/servo.yaml" />
  <group if="$(eval controller == 'spacemouse')">
    <node name="spacenav_node" pkg="spacenav_node" type="spacenav_node" />
    <node name="spacenav_to_twist" pkg="moveit_servo" type="spacenav_to_twist" output="screen" />
    <rosparam ns="moveit_servo">
    command_in_type: unitless
    </rosparam>
  </group>

  <node name="servo_server" pkg="moveit_servo" type="servo_server" output="screen" />

</launch>